### ISSUE TITLE
Uncaught Error

### ISSUE THREAD

**Issue by xntrik** [mozilla/seasponge#75]
Have spun up a fresh clone of seasponge. After installing dependencies and spinning up a local server (grunt serve). As soon as I hit the URL (automatically opened in Chrome) I get a blank screen. The console error is:

Uncaught Error: [$injector:unpr] Unknown provider: $$asyncCallbackProvider <- $$asyncCallback <- $animate <- $compile
This is in angular.js:68 (the last return statement).

```javascript
function minErr(module, ErrorConstructor) {
  ErrorConstructor = ErrorConstructor || Error;
  return function() {
    var SKIP_INDEXES = 2;

    var templateArgs = arguments,
      code = templateArgs[0],
      message = '[' + (module ? module + ':' : '') + code + '] ',
      template = templateArgs[1],
      paramPrefix, i;

    message += template.replace(/\{\d+\}/g, function(match) {
      var index = +match.slice(1, -1),
        shiftedIndex = index + SKIP_INDEXES;

      if (shiftedIndex < templateArgs.length) {
        return toDebugString(templateArgs[shiftedIndex]);
      }

      return match;
    });

    message += '\nhttp://errors.angularjs.org/1.4.3/' +
      (module ? module + '/' : '') + code;

    for (i = SKIP_INDEXES, paramPrefix = '?'; i < templateArgs.length; i++, paramPrefix = '&') {
      message += paramPrefix + 'p' + (i - SKIP_INDEXES) + '=' +
        encodeURIComponent(toDebugString(templateArgs[i]));
    }

    return new ErrorConstructor(message);
  };
}
```
Unsure if I've done something a little whack, or if perhaps with a fresh node/ruby/bower install there's been some changes in the dependencies.

**Comment by Glavin001** [comment]
I am not immediately sure what this issue could be. Potentially a dependency that broke something. Let me know if you discover anything else. Thanks!

**Comment by varwiz** [comment]
I get the same error too when i do a grunt --force

**Comment by kraghuveer** [comment]
I could resolve this error if I use angular 1.3.15 instead of angular 1.4.3.

Best regards,
Raghu.

**Comment by varwiz** [comment]
I replaced angular-animate.js in app/index.html because this file and bower injected angular had version mismatches. I guess replacing angular.js works too. Let me check.

**Comment by Frozenfire92** [comment]
So this is definitely a dependency out of date issue

I will look into it this weekend. If anyone gets around to it before me checkout out [package.json](https://github.com/mozilla/seasponge/blob/master/package.json) and update the dependency(ies?) there and submit a PR!

For reference was this on OSX, Linux?

**Comment by DinisCruz** [comment]
I just had the same error. I looks like this

![image](https://cloud.githubusercontent.com/assets/656739/10067753/d0c3d600-6296-11e5-8052-bc9d6c782108.png)

Is this the workaround

> You can replace the angular-animate file with the latest CDN link and do a build again.

**Comment by DinisCruz** [comment]
btw the updates branch is also affected

![image](https://cloud.githubusercontent.com/assets/656739/10067886/e04bb5ba-6297-11e5-9633-2a89a9c2821d.png)

**Comment by DinisCruz** [comment]
forcing it to 1.3.15 worked. 

here is my fix https://github.com/mozilla/seasponge/issues/79#issuecomment-142845405

**Comment by Glavin001** [comment]
Thanks for letting us know what worked for you, @DinisCruz.

**Comment by Frozenfire92** [comment]
I have merged #79 could @xntrik @DinisCruz @varwiz @kraghuveer test and reopen if the issue persists?
